src/main/java/com/klarna/hiverunner/sql/cli/DefaultPreProcessor.java

```diff
@@ -30,5 +30,7 @@ public String script(String script) {
     @Override
     public String statement(String statement) {
         return CommentUtil.stripFullLineComments(statement);
-    };
+    }
+
+    ;
```

Comment:
We don't need this `:` at all right? In this case I'd say remove it as the formatting makes it look very weird with it on its own line.

Reply:
I'll merge this now and then go take a look if it can be removed.